### ISSUE TITLE
Unknown Number of clusters

### ISSUE THREAD

**Issue by cdesch** [ayrat555/rock#3]
Is there a way to process a data set with an unknown number of clusters? If the data set had potentially multiple clusters within it, how would we process it to see each cluster?

**Comment by ayrat555** [comment]
@cdesch no, this algorithm requires you to pre-specify the number of clusters.

P.S I'm sorry for late reply.
